dss/api/subscriptions.py

```diff
@@ -3,35 +3,174 @@
 import json
 import re
 import typing
+from uuid import uuid4
 
 import iso8601
 import requests
 
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import NotFoundError
+from elasticsearch_dsl import Search
 from flask import jsonify, make_response, redirect, request
 from werkzeug.exceptions import BadRequest
 
-from .. import get_logger
-from .. import DSS_ELASTICSEARCH_INDEX_NAME, DSS_ELASTICSEARCH_DOC_TYPE, DSS_ELASTICSEARCH_QUERY_TYPE
-from .. import DSS_ELASTICSEARCH_SUBSCRIPTION_INDEX_NAME, DSS_ELASTICSEARCH_SUBSCRIPTION_TYPE
+from .. import (get_logger,
+                DSS_ELASTICSEARCH_INDEX_NAME, DSS_ELASTICSEARCH_DOC_TYPE, DSS_ELASTICSEARCH_QUERY_TYPE,
+                DSS_ELASTICSEARCH_SUBSCRIPTION_INDEX_NAME, DSS_ELASTICSEARCH_SUBSCRIPTION_TYPE)
 from ..blobstore import BlobNotFoundError
 from ..config import Config
+from ..error import DSSException, dss_handler
 from ..hcablobstore import FileMetadata, HCABlobStore
 from ..util.es import ElasticsearchClient, get_elasticsearch_index
 
 logger = get_logger()
 
 
+@dss_handler
 def get(uuid: str, replica: str):
-    pass
+    owner = request.token_info['email']
 
+    es_client = ElasticsearchClient.get(logger)
+    id_exists = es_client.exists(index=DSS_ELASTICSEARCH_SUBSCRIPTION_INDEX_NAME,
+                                 doc_type=DSS_ELASTICSEARCH_SUBSCRIPTION_TYPE,
+                                 id=uuid)
+    if not id_exists:
+        raise DSSException(requests.codes.not_found, "not_found", "Cannot find subscription!")
```

Comment:
You should update your entry in the swagger spec to expect this possible return value.